src/HermesWorker/index.js

```diff
@@ -234,6 +235,11 @@ class HermesWorker {
      */
     _onWorkerError(error) {
         console.error(error);
```

Comment:
is it usefull to log errors in workers ?

Reply:
If think no, maybe we should reject call promises instead